translations/canada.fr.yml

```diff
@@ -0,0 +1,39 @@
+fr:
+  new_years_day: Jour de l'an
+  new_years_day_observed: Jour de l'an (Observé)
+  family_day: Fête de la famille
+  louis_riel_day: Journée Louis Riel
+  islander_day: Fête des Insulaires
+  heritage_day: Fête du Patrimoine
+  st_patricks_day: Fête de la Saint-Patrick
+  good_friday: Vendredi saint
+  easter_monday: Lundi de Pâques
+  st_georges_day: Fête de la Saint-Georges
+  victoria_day: Fête de la Reine
+  national_patriots_day: Journée nationale des patriotes
+  national_aboriginal_day: Journée nationale des Autochtones
+  st_jean_baptiste_day: Fête nationale du Québec
+  st_jean_baptiste_day_observed: Fête nationale du Québec (Observé)
+  discovery_day: Jour de la Découverte
+  canada_day: Fête du Canada
+  canada_day_observed: Fête du Canada (Observé)
+  dominion_day: Jour du dominion
+  dominion_day_observed: Jour du dominion (Observé)
+  memorial_day: Jour de mémorial
+  memorial_day_observed: Jour de mémorial (Observé)
+  nunavut_day: Jour du Nunavut
+  nunavut_day_observed: Jour du Nunavut (Observé)
+  civic_holiday: Premier lundi d'août
+  british_columbia_day: Jour de la Colombie Britannique
+  new_brunswick_day: Jour du Nouveau Brunswick
+  saskatchewan_day: Jour du Saskatchewan
+  labour_day: Fête du Travail
+  queen_funeral: Funéraire de sa majesté la reine Elizabeth II
+  national_day_for_truth_and_reconciliation: Journée nationale de la vérité et de la réconciliation
+  thanksgiving: Action de grâce
+  remembrance_day: Jour du Souvenir
+  remembrance_day_observed: Jour du Souvenir (Observé)
+  christmas_day: Jour de Noël
+  christmas_day_observed: Jour de Noël (Observé)
+  boxing_day: Boxing Day
```

Comment:
Even in French, it's common to call this holiday by the English name of "Boxing Day"